src/main/java/com/feywild/feywild/data/BlockLootProvider.java

```diff
@@ -53,6 +53,13 @@ protected void setup() {
                 this.stack(ModItems.brilliantFeyGem)
         ));
 
+        this.drops(ModBlocks.feyGemBlockDeepSlate, true, this.first(
+                this.stack(ModItems.lesserFeyGem).with(this.random(0.6f)),
+                this.stack(ModItems.greaterFeyGem).with(this.random(0.6f)),
+                this.stack(ModItems.shinyFeyGem).with(this.random(0.6f)),
+                this.stack(ModItems.brilliantFeyGem)
+        ));
+
```

Comment:
Maybe add a method for fey gem drops now that we have three of them.

Or maybe a content suggestion: How about making the deepslate variant a little more rewarding. As deepslate takes longer to mine and thus is more annoying, we could give the palyer a tiny reward and set the chances to `0.5`, so it more likely propagates down to the rarer gems.

Reply:
Good idea. 
Changed chance drop for deepslate version. 
Added a description to patchouli as well. 
Added translation for the block.